trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-back-link-href.ts

```diff
@@ -0,0 +1,31 @@
+// "Recommended Security and Performance Max: 2048 CHARACTERS" - https://stackoverflow.com/a/48230425
+const MAX_URL_CHARACTERS = 2048;
+const BASE_URL_CHARACTER_ALLOWANCE = 150;
```

Comment:
Should these be constants?

Reply:
These were initially moved inline as a result of this comment, but these have now been moved back to constants as they're more readable and documentation can be attached to them rather than being inline within the function.